front/components/assistant/HelpDrawer.tsx

```diff
@@ -0,0 +1,211 @@
+import { createConversationWithMessage } from "@app/components/assistant/conversation/lib";
+import { SendNotificationsContext } from "@app/components/sparkle/Notification";
+import { GLOBAL_AGENTS_SID } from "@app/lib/assistant";
+import { useSubmitFunction } from "@app/lib/client/utils";
+
+const topPicks = [
+  {
+    title: "Creating Assistants",
+    href: "https://www.notion.so/dust-tt/Prompting-101-How-to-Talk-to-Your-Assistants-c0212513056a4af885e41706ee742dda",
+    icon: ArrowRightIcon,
+  },
+  {
+    title: "Managing Connections",
+    href: "https://www.notion.so/dust-tt/Dust-Library-1aa97619d56d4d9f9c21d7ce89d23dbb?pvs=4#4f7f969e52464e369f4cd453457d8059",
+    icon: ArrowRightIcon,
+  },
+  {
+    title: "Choosing a Model",
+    href: "https://blog.dust.tt/comparing-ai-models-claude-gpt4-gemini-mistral/",
+    icon: ArrowRightIcon,
+  },
+];
+
+function LinksList({
+  linksList,
+  title,
+}: {
+  linksList: {
+    title: string;
+    href: string;
+    icon?: ComponentType;
+    description?: string;
+  }[];
+  title?: string;
+}) {
+  return (
+    <Item.List>
+      {title && <Item.SectionHeader label={title} />}
+      {linksList.map((link, index) => (
+        <Item.Navigation
+          icon={link.icon}
+          label={link.title}
+          href={link.href}
+          key={index}
+          description={link.description}
+          hasAction={false}
+        />
+      ))}
+    </Item.List>
+  );
+}
+
+export function HelpDrawer({
+  owner,
+  user,
+  show,
+  onClose,
+}: {
+  owner: WorkspaceType;
+  user: UserType;
+  show: boolean;
+  onClose: () => void;
+}) {
+  const router = useRouter();
+  const sendNotification = useContext(SendNotificationsContext);
+
+  const { submit: handleHelpSubmit } = useSubmitFunction(
+    useCallback(
+      async (input: string, mentions: MentionType[]) => {
+        const inputWithHelp = input.includes("@help")
+          ? input
+          : `@help ${input.trimStart()}`;
+        const mentionsWithHelp = mentions.some(
+          (mention) => mention.configurationId === GLOBAL_AGENTS_SID.HELPER
+        )
+          ? mentions
+          : [
+              ...mentions,
+              { configurationId: GLOBAL_AGENTS_SID.HELPER } as AgentMention,
+            ];
+        const conversationRes = await createConversationWithMessage({
+          owner,
+          user,
+          messageData: {
+            input: inputWithHelp.replace("@help", ":mention[help]{sId=helper}"),
+            mentions: mentionsWithHelp,
+            contentFragments: [],
+          },
+        });
+        if (conversationRes.isErr()) {
+          sendNotification({
+            title: conversationRes.error.title,
+            description: conversationRes.error.message,
+            type: "error",
+          });
+        } else {
+          // We start the push before creating the message to optimize for instantaneity as well.
+          void router.push(
```

Comment:
So it serves as an intent to create a new conversation, but there is no logic at all happening in the drawer once the user submits its first message?

Reply:
Correct